clg/sum/calculate.go

```diff
@@ -1,3 +1,5 @@
+// Package subtract implements spec.CLG and provides the mathematical operation
```

Comment:
Good catch. I noticed this issue because of the failing CI. Travis executes the [project.check.sh](https://github.com/xh3b4sd/anna/blob/bfabf710374b8e2b0408544fc7a502d4d9228415/project.check.sh) which executes `golint`. This complained and I fixed it. See also https://github.com/xh3b4sd/anna/blob/master/.travis.yml#L12.